phpstan.neon

```diff
@@ -5,5 +5,4 @@ parameters:
 		- src
 		- tests
 	bootstrapFiles:
-		- bootstrap.php
```

Comment:
In that case, we're going to have to move the definition of
```php
define('SYSTEM_USER_UUID', '00000000-0000-0000-0000-000000000000');
```
to another location so that we can easily load it for phpstan (or replace the usages with a static call to a class or something)

Reply:
Alternatively we might be able to copy the `config.yml.dist` file to `config.yml` in github actions before running the tests, phpstan etc? Maybe that works even if it's just with dummy values. And then we could also use that later to configure our db connection for integration tests etc.